src/device.ts

```diff
@@ -11,6 +12,23 @@ const generateUUID = (): string => {
 };
 
 class Device {
+  static getPlatform(): string {
+    let userAgent = navigator.userAgent;
```

Comment:
Any idea how reliable this is? UserAgent parsing can have a lot of edge cases

Reply:
I think it's the best option for this use case. There's also `navigator.platform`, but that's deprecated. We could also let the developer decide (e.g., `Radar.trackVerified({ useDesktopApp: boolean = false })`), but that just shifts the problem to the developer